app/views/categories/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<% if @categories.empty?%>
+    <h1>Hello, currently you don't have any categories to choose from. You can create a new Category with by clicking
+    the button bellow.
+    </h1>
+<%else%>
+<div class="parent-cat">
+ <% @categories.each do |category|%>
```

Comment:
- [ ] Great job working on this capstone project. As per project requirements, when a user logs in, they are supposed to be presented with the categories page. At the moment, when a user logs it, the splash page remains. Kindly check your routes to ensure that the user is redirected to the correct page after logging in.
![image](https://github.com/IsmailChangezi/snap-scan/assets/62849872/d499133a-8d9b-48fa-82e1-186da159fb55)

- [ ] As per the screenshot above, ensure that the log in and sign up buttons are aligned centre. There is too much margin on the left